plugins/Eq/EqSpectrumView.h

```diff
@@ -144,7 +144,7 @@ class EqSpectrumView : public QWidget
 		setFixedSize( 400, 200 );
 		QTimer *timer = new QTimer(this);
 		connect(timer, SIGNAL(timeout()), this, SLOT(update()));
-		timer->start(100);
+		timer->start(10);
```

Comment:
@serdnab can you please address the questions above?  I agree with @Umcaruje 20 is fine.  It doesn't matter if your monitor is 200 fps, your eyes can't see it and for the purposes of a visual bouncing EQ 50 fps should be perfectly suitable.